CONTRIBUTING.md

```diff
@@ -0,0 +1,86 @@
+# Guidelines for Contribution
+
+<br>
+
+## How can I contribute in vscode-seneca-college project?
```

Comment:
I think grammatically, it would be better to say "contribute to" rather than "contribute in"